compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/JvmTerminatorGen.java

```diff
@@ -887,6 +888,15 @@ private void genStaticCall(BIRTerminator.Call callIns, PackageID packageID, int
                     packageID.orgName.getValue() + "/" + packageID.name.getValue());
             Name decodedMethodName = new Name(Utils.decodeIdentifier(methodName));
             BInvokableSymbol funcSymbol = (BInvokableSymbol) symbol.scope.lookup(decodedMethodName).symbol;
+            if (funcSymbol == null && JvmCodeGenUtil.isModuleInitializerMethod(decodedMethodName.value)) {
+                // moduleInit() and moduleStart() functions are not present in the BIR cache because they are generated
+                // in CodeGen phase. Therefore, they are not found inside the packageSymbol scope.
+                jvmClass = JvmCodeGenUtil.getModuleLevelClassName(packageID,
+                        JvmCodeGenUtil.cleanupPathSeparators(MODULE_INIT_CLASS_NAME));
+                methodDesc = MODULE_INITIALIZER_METHOD_DESC;
```

Comment:
We can directly use constant class

Reply:
Fixed in 8da28c04c1397a39613ef773b0f56c5989ce3016.